.github/Pull-Request-Template

```diff
@@ -0,0 +1,25 @@
+# PR Description
+Please include a brief description of the pull request you want to raise.
+# Description of tasks that were expected to be completed
+List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style
+- [ ] Create .....
+- [ ] Add .....
+- [ ] ......
+# Description of tasks that were expected to be completed and that are not yet completed
+- [ ] Create .....
+- [ ] Add .....
+- [ ] ......
+# How this may be tested?
+Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.
+1. run `npm i` to install all required dependencies
```

Comment:
update this file and use yarn as well @kunda4

Reply:
done